src/pages/ListPage.js

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from 'react';
+import { LocationsContext } from '../context/LocationsContext';
+import { sortLocationsByDistance } from '../helpers/helpersAPI';
+import { LocationList } from '../components/LocationList';
+import Spinner from '../components/Spinner';
+import ErrorMessage from '../components/ErrorMessage';
+
+const ListPage = () => {
+  const valueLocations = useContext(LocationsContext);
+  const { locations, status, error } = valueLocations;
+  const locationsSorted = sortLocationsByDistance(locations);
+
+  return (
+    <>
+      <h1 className="list__title">List</h1>
+      <div className="view__content">
+        {status === 'loading' && <Spinner />}
```

Comment:
When you’ve got a conditional rendering, it’s a good habit to use ternaries😄 . You could check why this kind of pattern is recommended here: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Thank you, good view! We will change to ternaries.